source/less/components/video.less

```
@@ -90,20 +98,96 @@
 .video__button {
   .reset-button;
   position: absolute;
+  top: 133px;
+  left: 50%;
+  width: 100px;
+  height: 76px;
+  margin-left: -50px;
+  padding: 0;
+  background-image: url("../img/video/play.svg");
```

```suggestion
  background-image: url("/img/video/play.svg");
```

поменяй все урлы в css на абсолютные, тк когда есть сборка, то есть вероятность, что пути в сборке будут не так относительны друг друга как в сырцах, так что лучше абсолютные пути. Сборщик их дописывает до нужных